pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
 	"networkx >= 2.6",
 	"numpy ~= 1.24",
 	"opencv-python ~= 4.5",
-	"pegen >= 0.2",
+	"pegen @ git+https://github.com/we-like-parsers/pegen.git",
```

Comment:
Hmm, this seems a bit risky: does `pegen` make any claims about the stability of their `main` branch? We could ask when they plan to make a new release, and if it isn't imminent we could pin to a specific commit temporarily.

Reply:
Strongly agree, let's discuss offline.

Reply:
Fixed with new Pegen 0.3.0.